runtime/altair/src/lib.rs

```diff
@@ -870,7 +865,8 @@ construct_runtime!(
 		Identity: pallet_identity::{Pallet, Call, Storage, Event<T>} = 67,
 		Vesting: pallet_vesting::{Pallet, Call, Storage, Event<T>, Config<T>} = 68,
 		Treasury: pallet_treasury::{Pallet, Call, Storage, Config, Event<T>} = 69,
-		// Uniques: pallet_uniques::{Pallet, Call, Storage, Event<T>} = 70,
+		Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 70,
+		Uniques: pallet_uniques::{Pallet, Call, Storage, Event<T>} = 72,
```

Comment:
No pallet 71?

Reply:
It is defined a bit above, and seems that for some reasons there is an order dependency.

Reply:
@mustermeiszer I just compiled successfully changing the order moving down pallet 71 to its ordered place, do you why you added it up there before?

Reply:
When you start the chain the storage will not be present and we will not be to inject candidates from the collator-selection pallet. As mentioned in the comment.

Reply:
Okay, then leaving it up there